fdbserver/workloads/ChangeConfig.actor.cpp

```diff
@@ -71,8 +83,10 @@ struct ChangeConfigWorkload : TestWorkload {
 				TraceEvent("WaitForReplicasExtra").log();
 				wait(waitForFullReplication(db));
 				TraceEvent("WaitForReplicasExtraEnd").log();
+				existingDB = true;
 			}
-			wait(success(ManagementAPI::changeConfig(db.getReference(), self->configMode, true)));
+			std::string configMode = self->getConfigMode(self->configMode, existingDB);
```

Comment:
this has to be a `state` variable to avoid stack-use-after-free bug.

Reply:
We discussed offline. This shouldn't be an issue, as the argument to the actor function is copied into the member variable in the ActorXXXState class.